scripts/Secured chat machine/README.md

```diff
@@ -0,0 +1,64 @@
+<h1 align="center"> Chat Machine  </h1>
+
+<h2 align="right">
+
+![python](https://forthebadge.com/images/badges/made-with-python.svg)
+
+</h2>
+
+This is WebAPP which provides Secured Chat messaging feaure with Encryption ans decyption security tool. 
+
+Base64 encryptions are scheme used in encryptions and decryption
+
+
+## __ABOUT__
+**ChatMachine** is a only suggest how can we achieve our End-to-End encyotion schems thats leads to secures messaging in python application that helps user to maintain our privacy from hackers and thirs party applications 
```

Comment:
Fix the typos on this line. Also this is not end-to-end encryption fix that as well.